.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -1,21 +1,45 @@
 ---
-name: Report a bug
-about: A bug you encountered with the sentry_flutter package.
+name: Bug report
+about: Create a report to help us improve
+
 ---
 
-<!-- Continue to report a bug on the **Sentry Flutter SDK**.
-For bug reports for the core sentry package, please use the the sentry-dart issue tracker:
-http://github.com/getsentry/sentry-dart/issues  -->
+_Platform:_
+- [ ]  Dart
+- [ ]  Flutter Android or iOS
+- [ ]  Flutter Web
+
+_IDE:_
+- [ ]  VSCode
+- [ ]  IntelliJ/AS
+- [ ]  XCode
+- [ ]  Other, which one?
 
-1. Please describe the issue and how to reproduce it, or link to a small reproducible application:
+_split-debug-info and obfuscate (Flutter Android or iOS) or CanvasKit (Flutter Web):_
+- [ ]  Enabled
+- [ ]  Disabled
 
+_Platform installed with:_
+- [ ] pub.dev
+- [ ] GitHub
 
+Output of the command `flutter doctor` below::
```

Comment:
Shouldn't it be `flutter doctor -v`? Also double colons at the end.

Reply:
good idea, I just copied over the old template, thanks @ueman :)